### PR TITLE
Multiple fixes to types/chroma-js/index.d.ts

### PR DESCRIPTION
**Pull request by NaridaL**
* Fixed bezier method
* typed string parameters to be string unions
* added out parameter to Scale.colors
* changed color types to number tuples instead of arrays
* added types/chroma-js/tslint.json

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: http://gka.github.io/chroma.js/ 
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by NaridaL** [comment]
Not sure what's going on with the CI build. I copied the tslint from another dir. My IDE ran it cleanly.

File: types/chroma-js/index.d.ts [hunk 1]
```diff
@@ -31,21 +44,21 @@ declare namespace Chroma {
          * @param a
          * @param b
          * @param c
-         * @param colorSpace The color space to use (one of "rgb", "hsl", "hsv", "lab", "lch", "gl"). Defaults to "rgb".
+         * @param colorSpace The color space to use. Defaults to "rgb".
          * @return the color object.
-        */
-        (a: number, b: number, c: number, colorSpace?: string): Color;
+         */
+        (a: number, b: number, c: number, colorSpace?: keyof ColorSpaces): Color;
```
Comment (mhegazy) [inline types/chroma-js/index.d.ts]: since you are using `keyof` you need `// TypeScript Version: 2.1`

Comment (mhegazy) [inline types/chroma-js/index.d.ts]: that is what is causing the CI test failures.

File: types/chroma-js/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Chroma.js v1.1.1
+// Type definitions for Chroma.js v2.0.0
 // Project: https://github.com/gka/chroma.js
 // Definitions by: Sebastian Brückner <https://github.com/invliD>, Marcin Pacholec <https://github.com/mpacholec>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
Comment (andy-ms) [inline types/chroma-js/index.d.ts]: `// TypeScript Version: 2.1` comment should go immediately beneath this line.

**Comment by NaridaL** [comment]
Yeah sorry about that. Went through now.

File: package.json [hunk 3]
```diff
@@ -21,7 +21,10 @@
     "lint": "dtslint types"
   },
   "devDependencies": {
```
Comment (andy-ms) [inline package.json]: Don't change this file.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**